loleaflet/src/control/Control.TopToolbar.js

```diff
@@ -99,7 +99,8 @@ L.Control.TopToolbar = L.Control.extend({
 			{type: 'button',  id: 'undo',  img: 'undo', hint: _UNO('.uno:Undo'), uno: 'Undo', disabled: true, mobile: false},
 			{type: 'button',  id: 'redo',  img: 'redo', hint: _UNO('.uno:Redo'), uno: 'Redo', disabled: true, mobile: false},
 			{type: 'button',  id: 'formatpaintbrush',  img: 'copyformat', hint: _UNO('.uno:FormatPaintbrush'), uno: 'FormatPaintbrush', mobile: false},
-			{type: 'button',  id: 'reset',  img: 'deleteformat', hint: _UNO('.uno:ResetAttributes', 'text'), uno: 'ResetAttributes', mobile: false},
+			{type: 'button',  id: 'resetdefault',  img: 'deleteformat', hint: _UNO('.uno:SetDefault', 'presentation', 'true'),hidden: true, uno:'SetDefault', mobile: false},
+			{type: 'button',  id: 'reset',  img: 'deleteformat', hint: _UNO('.uno:ResetAttributes', 'text'),hidden: true, uno: 'ResetAttributes', mobile: false},
```

Comment:
I would leave 'reset' in its place and add 'resetdefault' as second one, below it. This way it shows in the diff that you added 'hidden' for the 'reset' and then added a completely new entry 'resetdefault'.

Also, it might be a better idea to label the Impress-specific one as 'resetimpress' or something like that. 'resetdefault' sounds generic and misleading.

Finally, keep the formatting as consistent as possible: you need a space before 'hidden'.